stories/features/reboot.fmf

```diff
@@ -24,10 +24,12 @@ description: |
     option of the reboot script, e.g. ``tmt-reboot -t 3600`` to
     wait for up to an hour for the guest to come back up.
 
-    On machines booted in UEFI mode, by default, the *BootNext*
-    property shall be set to the value of *BootCurrent*. The
-    ``-e`` option of the reboot script can be used to prevent
-    this from being set, e.g. ``tmt-reboot -e``.
+    On Beaker machines booted in UEFI mode, by default, the
+    ``BootNext`` property shall be set to the value of ``BootCurrent``.
+    If ``BootCurrent`` cannot be found, use the value of file
+    ``/root/EFI_BOOT_ENTRY.TXT`` for ``BootNext``. The ``-e``
+    option of the reboot script can be used to force this behaviour,
+    e.g. ``tmt-reboot -e``.
```

Comment:
The wording is still confusing. What is "this behaviour" in this context? From what I can read in the script the workflow is quite different:
- By default the efibootmgr is not tinkered with
- If there is a `/root/EFI_BOOT_ENTRY.TXT` we will tinker with it
- If the user uses `tmt-reboot -e` we will also tinker with it

And the tinkering involves:
- Try to get `BootCurrent`
- If we didn't find `BootCurrent` fallback to `/root/EFI_BOOT_ENTRY.TXT` (why is the order not the other way around?)
- Make sure we are booting from the "intended" `BootCurrent`